src/deploy/osp_deployer/director.py

```diff
@@ -706,17 +709,44 @@ def configure_dhcp_server(self):
     def setup_networking(self):
         logger.debug("Configuring network settings for overcloud")
 
+        if self.settings.enable_ovs_dpdk:
```

Comment:
It looks like these 3 lines are unnecessary because both mode and compute_file_name are set below on lines 731/732.

Reply:
Agree. Removed